packages/perseus/src/widgets/interactive-graphs/graphs/quadratic.test.tsx

```diff
@@ -328,3 +295,74 @@ describe("describedQuadraticGraph interactive elements", () => {
         );
     });
 });
+
+describe("getQuadraticCoefficients", () => {
```

Comment:
I renamed this as it was only testing the getQuadraticCoefficients function.

Reply:
Thank you for this comment! I'm confused why it can't figure out the body of this is the same as before.